### PR TITLE
Update PCB's

### PR DESCRIPTION
**Pull request by jamie-54**
## Main Changes

Made changes to the PCB's as outlined in issue #135 and updated the associated readme. The new updated PCBs can be easily seen [here](https://github.com/QUB-ASL/bzzz/tree/feature/135-PCBs/design/PCBs).

## Associated Issues

- Addresses #135 

### Raspberry Pi board

RPi board pcb

RPi board schematic

### Sensor board

sensor board pcb

sensor board schematic

### ESP32 board (with two IMUs)

esp board pcb

esp board schematic

### CODE REVIEW

**Comment by alphaville** [comment]
@jamie-54 What's the status of this? Is it ready to be reviewed?

**Comment by jamie-54** [comment]
Nearly, going to update now with UART2 included for TeraBee Evo time of flight sensor

File: design/PCBs/PI_PCB/PI_PCB_HDMI_x1.png
Comment (alphaville) [inline design/PCBs/PI_PCB/PI_PCB_HDMI_x1.png:1]: @jamie-54, this is still the old version with the HDMI

File: design/PCBs/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+## ESP board with 2 IMU's
+
+<img src="ESP_PCB/ESP_PCB_2_IMU_x1.png">
+
+## ESP schematic 2 IMU's
+
+<img width="600" src="ESP_PCB/ESP_SCH_2_IMU.png">
+
+## Pi board with HDMI
```
Comment (alphaville) [inline design/PCBs/README.md:28]: This needs to be updated (HDMI -> ribbon)

**Review by alphaville — COMMENTED** [review]
In principle everything looks good to me. Just a few comments:

1. The README file needs to be updated with the new schematics and PCBs and we should remove any files that are not relevant any more (e.g., the ones using the HDMI)
2. Can we add a small logo on the boards somewhere if there is space?
3. Can you add a link to the ribbon plug that will be used on the boards?
4. Would it be better to have a ground in the same header as the ESCs? What do you think?
5. On the ESP32 board can we print something on the PCB to indicate on which side the ESP goes and in what direction? In general, the more information we can have printed on the board, the better.
6. Now that we have finalised the boards, let's order any components we need.

@Yuanbwcx and @pdavid747, have a look as well - the more people check this before we send it out for printing the better. Let us know if any changes are needed to accommodate the needs of your projects.

**Comment by jamie-54** [comment]
> In principle everything looks good to me. Just a few comments:
> 
> 1. The README file needs to be updated with the new schematics and PCBs and we should remove any files that are not relevant any more (e.g., the ones using the HDMI)
> 2. Can we add a small logo on the boards somewhere if there is space?
> 3. Can you add a link to the ribbon plug that will be used on the boards?
> 4. Would it be better to have a ground in the same header as the ESCs? What do you think?
> 5. On the ESP32 board can we print something on the PCB to indicate on which side the ESP goes and in what direction? In general, the more information we can have printed on the board, the better.
> 6. Now that we have finalised the boards, let's order any components we need.
> 
> @Yuanbwcx and @pdavid747, have a look as well - the more people check this before we send it out for printing the better. Let us know if any changes are needed to accommodate the needs of your projects.

1. I'll fix this at the end after the other issues are sorted.
2. Yea I've added logos to Pi board and sensor board. ESP board is too small to add it anywhere it will be seen.
3. Yea I can add a BOM somewhere (either in read.me or separate file)
4. There's enough room so I can add it and we can decide later if we want to use it or not.
5. The yellow stuff is the top silkscreen layer so ESP32 outline is already there with the USB port to indicate direction but might label some pins 
6. think most if not all components or ordered or on the next order list but I'll check

**Comment by alphaville** [comment]
@pdavid747 and @Yuanbwcx will you be able to review this today?

**Review by alphaville — COMMENTED** [review]
@jamie-54 the buzzer on the RPi board will be placed downwards (toward the Pi)?

**Comment by jamie-54** [comment]
> @jamie-54 the buzzer on the RPi board will be placed downwards (toward the Pi)?

No think it will have to be placed upwards as that part of the board is above the USB ports, but should probably move the buzzer closer to the top corner to make sure sure it avoids the USB going to the ESP32

**Comment by alphaville** [comment]
> > @jamie-54 the buzzer on the RPi board will be placed downwards (toward the Pi)?
> 
> No think it will have to be placed upwards as that part of the board is above the USB ports, but should probably move the buzzer closer to the top corner to make sure sure it avoids the USB going to the ESP32

Is there a way to get a figure showing the two boards one on top of the other?

**Comment by jamie-54** [comment]
> > > @jamie-54 the buzzer on the RPi board will be placed downwards (toward the Pi)?
> > 
> > 
> > No think it will have to be placed upwards as that part of the board is above the USB ports, but should probably move the buzzer closer to the top corner to make sure sure it avoids the USB going to the ESP32
> 
> Is there a way to get a figure showing the two boards one on top of the other?

I’ll take a look later

**Comment by jamie-54** [comment]
> > > > @jamie-54 the buzzer on the RPi board will be placed downwards (toward the Pi)?
> > > 
> > > 
> > > No think it will have to be placed upwards as that part of the board is above the USB ports, but should probably move the buzzer closer to the top corner to make sure sure it avoids the USB going to the ESP32
> > 
> > 
> > Is there a way to get a figure showing the two boards one on top of the other?
> 
> I’ll take a look later

I've moved the buzzer over. Here is the 2 boards on top of each other, sorry if its hard to make out

PCB_PCB_DRONE_RPI_SHIELD_FPC_2023-11-21

**Review by alphaville — APPROVED** [review]
LGTM

**Pull request merged**